src/controllers/controllerdebug.h

```diff
@@ -23,10 +25,24 @@ class ControllerDebug {
         s_enabled = false;
     }
 
+    static void enableTesting() {
+        s_enabled = true;
+        s_testing = true;
+    }
+
+    static ControlFlags shouldAssertForInvalidControlObjects() {
```

Comment:
The function name is a question that can be answered with true or false but not with individual flags.
I would prefer to make this bool, because the flags are an implementation detail of code out of sight. 
Alternative we need a new name.